File: C/EX14.cpp
#include <conio.h>
#include <stdlib.h>
#include <stdio.h>
#include <time.h> 
#include <string.h>
#include <math.h>
#include <locale.h>
#include <ctype.h>

int a,b,c,x = 0;

int main()
{
	
	
			printf(" \n Digite o primeiro valor:\n");
			scanf("%d",&a);
			printf(" \n Digite o segundo valor:\n");
			scanf("%d",&b);
			printf(" \n Digite o terceiro valor:\n");
			scanf("%d",&c);
			
			if(a>b)
			{
				if(a>c)
				{
					printf(" \n O maior valor inserido foi:%d",a);
				}
				else 
				{
					printf(" \n O maior valor inserido foi:%d",c);	
				}
				
			}
			
				if(b>a)
			{
				if(b>c)
				{
					printf(" \n O maior valor inserido foi:%d",b);
				}
				else 
				{
					printf(" \n O maior valor inserido foi:%d",c);	
				}
			}

	
}

File: Python/EX14.py
valor1 = int(input('digite um valor: '))
valor2 = int(input('digite outro valor: '))
valor3 = int(input('digite outro valor: '))

if(valor1>valor2):
    if(valor1>valor3):
        print("O maior valor e: ",valor1)
        
    else:
        print('O maior valor e :',valor3)

else:
    if(valor2>valor3):
        print('O maior valor e: ', valor2)
    
    else:
        print('O maior valor e: ', valor3)
